src/Qt/PlayerPrivate.cpp

```diff
@@ -109,15 +109,14 @@ namespace openshot
             const auto render_time = double_micro_sec(time2 - time1);
 
             // Calculate the amount of time to sleep (by subtracting the render time)
-//            auto sleep_time = duration_cast<micro_sec>( (frame_duration - render_time));
-            auto correction_factor = video_frame_diff * video_frame_diff * video_frame_diff * video_frame_diff * video_frame_diff;
-//            auto sleep_time_no_correction = duration_cast<micro_sec>((frame_duration - render_time));
+            auto correction_factor = video_frame_diff * video_frame_diff * video_frame_diff;
```

Comment:
Can we use `std::powf` instead? https://en.cppreference.com/w/cpp/numeric/math/pow